checker/tests/nullness/Issue2432b.java

```diff
@@ -0,0 +1,42 @@
+// Additional test cases for issue 2432 (unrelated to poly annotations)
+// https://github.com/typetools/checker-framework/issues/2432
+
+import java.util.ArrayList;
+import java.util.List;
+import org.checkerframework.checker.nullness.qual.PolyNull;
+
+class Issue2432b {
+    void objectAsTypeArg() {
+        List<Object> objs = new ArrayList<>();
+        // no error
+        Object[] objarray = objs.toArray();
+    }
+
+    void myClassAsTypeArg() {
+        MyClass<Object> objs = new MyClass<>();
+        Object[] objarray = objs.toArray();
+        // no error
```

Comment:
Can you add a few tests that cause expected errors?

Reply:
It's somehow hard because this is unrelated to the nullness type system itself, but rather a internal error in handling the type variable which only produce false positive. Real negative in this case should be captured by javac I think.